Makefile

```
@@ -15,7 +15,7 @@ install-hooks:
 
 install-fhir-validator:
 	mkdir -p bin
-	test -f bin/org.hl7.fhir.validator.jar || curl -L https://github.com/hapifhir/org.hl7.fhir.core/releases/download/5.6.42/validator_cli.jar > bin/org.hl7.fhir.validator.jar
+	test -f bin/org.hl7.fhir.validator.jar || curl -L https://github.com/hapifhir/org.hl7.fhir.core/releases/download/6.0.22/validator_cli.jar > bin/org.hl7.fhir.validator.jar
```

I was unable to change this back to `latest`, I don't think that URL exists anymore

---

Because, for some reason, you need to reverse the order... This should work...
```suggestion
	test -f bin/org.hl7.fhir.validator.jar || curl -L https://github.com/hapifhir/org.hl7.fhir.core/releases/latest/download/validator_cli.jar > bin/org.hl7.fhir.validator.jar
```

---

Bah. Done